capstone-report.md

```diff
@@ -73,31 +81,32 @@ The augmented data is then used to train a CNN model using transfer learning. Th
 height: 150px
 name: acc_chart
 ---
-The results of accuracy.
+All four models were tested on a holdout sample to produce these accuracy results. 
 ```
 
 ```{figure} image/model_recall_bar_chart.png
 ---
 height: 150px
 name: recall_chart
 ---
-The results of recall.
+All four models were tested on a holdout sample to produce these recall results. 
 ```
 
 
-The model that utilizes the densenet architecture appears to be the best performing model for our dataset. Furthermore, we conducted a manual analysis of tricky negative and positive images to test model performance using an interactive interface. This application allowed us to compare how confident the models were when misclassifying cases and showcased how well DenseNet was doing. 
+The model that utilizes the DenseNet architecture appears to be the best performing model for our dataset. Furthermore, we conducted a [manual analysis](https://github.com/UBC-MDS/capstone-gdrl-lipo/blob/master/notebooks/model_decision_making.ipynb) of tricky negative and positive images to test model performance using an interactive interface. This application allowed us to compare how confident the models were when misclassifying cases and showcased how well DenseNet was doing. 
+
+In an effort to reduce the high confidence levels for misclassifications, a problem known as overfitting, we considered adding dropout layers in our CNN structure. In this case, overfitting occurs as a result of having trained our models on such a small training set. Another way to reduce overfitting is to average the predictions from all four models, called an ensemble. However, an ensemble would not be feasible as it requires lots of resources such as enough CPU to train the models. Although the dropout layers did manage to reduce overfitting in the models, the overall reduced accuracy was not significant enough to implement those layers in our optimal model. To see this exploration, click [here](https://github.com/UBC-MDS/capstone-gdrl-lipo/blob/master/notebooks/densemodels-ax-dropout-layers.ipynb).
```

Comment:
Do we have the correct understanding of overfitting here? My understanding is that overfitting is when the model models the training set too closely and not high confidence levels for misclassification. High confidence levels for misclassification may result from overfitting but they may also be due to other errors (in our case wasn't it mostly because we had the wrong test-valid-train set?)

Reply:
Tagging @larahabashy here who would be the best suited person to address this tomorrow.

Reply:
@elabandari ! This feedback has been incorporated by @larahabashy and myself!